alphagov/paas-cf
# [#159733111] Remove compose broker 🔥

What
----

This removes the compose broker from our deployment.

It includes a couple of temporary tasks to delete the broker, and to remove the secrets from the state bucket. These can be removed once they've run everywhere.

While working on this, I discovered that there was a fair amount of duplication in the list of secrets to be preserved when rotating. I've therefore taken the opportunity to de-dupe and alphabetise this list.

How to review
-------------

* Run the pipeline from this branch
* Verify all traces of the broker are gone
* Run the pipeline again to confirm that the removal is idempotent.

Who can review
--------------

Not me.
